core/controllers/profile.py

```diff
@@ -170,7 +214,7 @@ def put(self):
         """Handles PUT requests."""
         update_type = self.payload.get('update_type')
         data = self.payload.get('data')
-
+        show_bulk_email_signup_message = False
```

Comment:
bulk_email_signup_message_should_be_shown perhaps, to avoid starting with a verb

Reply:
Done